ValetTests/ValetTests.m

```diff
@@ -110,8 +110,11 @@ - (void)test_initialization_invalidArgumentsCauseFailure;
 {
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"" accessibility:VALAccessibilityAlways]);
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"test" accessibility:0]);
-    XCTAssertNil([[VALSynchronizableValet alloc] initWithIdentifier:@"test" accessibility:VALAccessibilityWhenPasscodeSetThisDeviceOnly]);
```

Comment:
I moved to `VALAccessibilityWhenUnlockedThisDeviceOnly` so I wouldn't have to wrap that line of test in `#if__IPHONE_8_0 || __MAC_10_10`